WalletWasabi.Tests/UnitTests/WabiSabi/Backend/PhaseStepping/StepTransactionSigningTests.cs

```diff
@@ -30,7 +30,7 @@ public async Task EveryoneSignedAsync()
 		var (key1, coin1, key2, coin2) = WabiSabiFactory.CreateCoinKeyPairs();
 
 		var mockRpc = WabiSabiFactory.CreatePreconfiguredRpcClient(coin1.Coin, coin2.Coin);
-		using Arena arena = await WabiSabiFactory.CreateAndStartArenaAsync(cfg, mockRpc);
+		using Arena arena = await ArenaBuilder.From(cfg).With(mockRpc.Object).CreateAndStartAsync();
```

Comment:
One can use the new overload imo:

```suggestion
		using Arena arena = await ArenaBuilder.From(cfg).With(mockRpc).CreateAndStartAsync();
```